third_party/psychopy/configuration.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+Copyright 2016-2022 The FEAGI Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================
+"""
+
+app_name = 'gazebo'
```

Comment:
Currently this works only when name is Gazebo but we need to rename that to a better name, perhaps "robot"

Reply:
But we will have to change it on FEAGI side as well along with all othe applications